pipeline/transformer/transformer_mets/src/test/scala/uk/ac/wellcome/platform/transformer/mets/fixtures/MetsGenerators.scala

```diff
@@ -0,0 +1,23 @@
+package uk.ac.wellcome.platform.transformer.mets.fixtures
+
+import uk.ac.wellcome.models.work.internal.License
+
+trait MetsGenerators {
+  def metsXmlWith(recordIdentifier: String, license: License) =
+    s"""|<?xml version='1.0' encoding='utf-8'?>
+       |<mets:mets xmlns:mets="http://www.loc.gov/METS/" xmlns:mods="http://www.loc.gov/mods/v3">
+       |  <mets:dmdSec ID="DMDLOG_0000">
+       |    <mets:mdWrap MDTYPE="MODS">
+       |      <mets:xmlData>
+       |        <mods:mods>
+       |          <mods:recordInfo>
+       |            <mods:recordIdentifier source="gbv-ppn">$recordIdentifier</mods:recordIdentifier>
+       |          </mods:recordInfo>
+       |          <mods:accessCondition type="dz">${license.id.toUpperCase}</mods:accessCondition>
+       |        </mods:mods>
+       |      </mets:xmlData>
+       |    </mets:mdWrap>
+       |  </mets:dmdSec>
+       |</mets:mets>
+       |""".stripMargin
```

Comment:
Scala has XML literals, so could use that here. Going to be removed at some time in the future in Scala 3 AFAIK.